src/actions/authActions.js

```diff
@@ -74,3 +74,9 @@ export const createUser = (userData, history) => async (dispatch) => {
     });
   }
 };
+
+export const loginViaSocial = (brand) => async dispatch => {
+  dispatch({ type: null });
+  localStorage.setItem('socialLogin', true);
```

Comment:
Great Job @encodedBicoding. I see a token in the URL when I'm redirected to our platform. But there is none set in the local storage and therefore, the `login` is not successful. Please take a look.

Reply:
that will not work because of the link from the backend. when that PR is merged, all will be well. Thanks